mech_client/cli.py

```diff
@@ -50,7 +50,7 @@ def cli() -> None:
 
 
 @click.command()
-@click.argument("prompt")
+@click.option("--prompts", type=str, multiple=True, help="Prompt to send for request")
```

Comment:
Do you think it would make sense to change the help to something like:

> One or more prompts to send as a request. Can be repeated.

and similar for the tools below?